.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true
+  },
+  extends: ['standard'],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly'
+  },
+  parserOptions: {
+    ecmaVersion: 2018
+  },
+  parser: "babel-eslint",
```

Comment:
ESLint by itself doesn't support private variables, need to use this external parser. (See [here](https://stackoverflow.com/a/57393894))